src/main/java/org/opensearch/geospatial/stats/StatsNodeRequest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.stats;
+
+import java.io.IOException;
+
+import org.opensearch.action.support.nodes.BaseNodeRequest;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+
+public class StatsNodeRequest extends BaseNodeRequest {
+
+    private StatsRequest request;
+
+    public StatsNodeRequest() {
+        super();
```

Comment:
This would make request null. Is there a reason for this constructor? I remember somewhere there may have been a required no-arg constructor but I cant remember if it was here.

Reply:
Good catch. I removed the default constructor. The one that needs default constructor is TransportAction